.eslintrc.js

```diff
@@ -1,20 +1,43 @@
 module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    'airbnb',
+    'prettier',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: './tsconfig.json',
-    tsconfigRootDir: './',
-    // TODO: we need this because of an issue with @typescript-eslint/parser: https://github.com/typescript-eslint/typescript-eslint/issues/864
-    createDefaultProgram: true,
```

Comment:
Why you removed this lines? We need them if we want IDE support. It is a eslint bug and this is the only way to make it check newly created files without restarting VSCode.

Reply:
This has been added back.